src/Button/index.scss

```diff
@@ -73,6 +74,7 @@
   font-weight: var(--font-weight-semibold);
   border-radius: 0;
   justify-content: start;
+  min-height: unset;
```

Comment:
The deprecated `PlainButton` did not have a min-height and neither should `Button type="plain"`